### PR TITLE
[Release] sdk/resourcemanager/iotcentral/armiotcentral/2.0.0-beta.1

### PR DESCRIPTION
**Pull request by Alancere**

### CODE REVIEW

File: sdk/resourcemanager/iotcentral/armiotcentral/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,88 @@
 # Release History
 
+## 2.0.0-beta.1 (2022-05-24)
+### Breaking Changes
+
+- Function `Resource.MarshalJSON` has been removed
```
[the remark this answers is not in the document]
Comment (tadelesh) [reply sdk/resourcemanager/iotcentral/armiotcentral/CHANGELOG.md]: I agree. Tool changed and PR updated.

**Comment by tadelesh** [comment]
Breaking changes from swagger: https://github.com/Azure/azure-rest-api-specs/pull/17439. All the model breaking is from changing self-defined model to common model.

**Comment by tadelesh** [comment]
Wait for codegen update.

**Comment by jhendrixMSFT** [comment]
Which codegen change is this one waiting for?

**Comment by tadelesh** [comment]
> Which codegen change is this one waiting for?

@jhendrixMSFT I'm waiting for the change of always generating marshallers/unmarshallers method. Have we made the decision for the change?

**Comment by jhendrixMSFT** [comment]
Will close on the un/marshaller question today.

**Review by tadelesh — CHANGES_REQUESTED** [review]
@Alancere You need to re-generate from the last released version, not last commit.

**Review by tadelesh — APPROVED** [review]

**Pull request merged**